src/main/java/uk/ac/cam/cl/dtg/segue/dao/users/PgUsers.java

```diff
@@ -15,6 +15,8 @@
  */
 package uk.ac.cam.cl.dtg.segue.dao.users;
 
+import com.fasterxml.jackson.core.JsonProcessingException;
```

Comment:
Easiest to see the changes to this file in the two commits to it. One is just a clean-up commit for consistency. The actual changes are found in one commit which deal with storing and retrieving the new database field.